.circleci/config.yml

```diff
@@ -143,12 +143,9 @@ commands:
   framework-tests:
     steps:
       - build
-      - run:
-          name: Installing dependencies
-          command: npm install
       - run:
           name: Running Framework Tests
-          command: npm run test:framework
+          command: scripts/circleci-run-tests.sh
```

Comment:
Just curious, why can't we run framework tests directly from here?

Reply:
because framework tests require account creds, and the creds are setup in `scripts/circleci-run-tests.sh` (they are different for stage / prod / dev)

Reply:
@charliesantos As Makarand said, we were still hitting credential errors when running from npm script even with the changes. So we had to do what we did for integration tests in order for us to pass in stage/dev creds.